### ISSUE TITLE
Can't login

### ISSUE THREAD

**Issue by Mazeee1** [DaanGamesDG/PaperPlane#78]
Hi.
Whenever I try to login to the CDN, it won't accept my password (or username). I've tried both the password I set in the .env file and the one I set while setting up the user (because I don't know which one works).

I'm not the most experienced person in this, so I've probably just done something wrong.

![image](https://user-images.githubusercontent.com/79224065/149321510-3fac3a38-d961-4d3d-a37d-15295c647197.png)

**Comment by DaanGamesDG** [comment]
Oh, that’s a problem. Can you give me a screenshot of the .env file (you exclude the password there), as well as a screenshot of the console when you press the login button.

to access the console, you have to use: crtl + shift + i, from there press the arrow in the top right corner and click on console.

**Comment by Mazeee1** [comment]
Here's my .env file:
![env file](https://user-images.githubusercontent.com/79224065/149327059-9ca97a7c-d568-4f16-b29b-daa1d8a578f9.png)

The text that I've put a red border around comes up when I login:
![console](https://user-images.githubusercontent.com/79224065/149327329-c8cb0ced-b7f4-4a65-8381-cdf5d7c2b2ae.png)

**Comment by DaanGamesDG** [comment]
Ah I see the problem, you forgot to add “https://” to your variable, you need to change the value of NEXT_PUBLIC_DOMAIN to “https://cdn.mazer800.xyz”

don’t forget to restart your server after that

**Comment by Mazeee1** [comment]
I've restarted my server then started the pm2 process but it still won't let me login. It looks like it's because of the same reason as well.

**Comment by DaanGamesDG** [comment]
This is probably because you did not add the “--update-env” flag to it, if you don’t add it, it will not update the env variables

**Comment by Mazeee1** [comment]
Ah, thanks.
Should it go on the outside or the inside of the quotation marks? Sorry if I'm being annoying.

![image](https://user-images.githubusercontent.com/79224065/149329768-060ef9f5-51ef-4512-89b8-a61f68f055bb.png)
![image](https://user-images.githubusercontent.com/79224065/149329772-71272aeb-d53b-4e24-88d3-eb089536f79e.png)

**Comment by Mazeee1** [comment]
Ah, thanks.
Should I put it on the outside or the inside of the quotation marks? Sorry if I'm being annoying

![1](https://user-images.githubusercontent.com/79224065/149330000-85266b9a-b5f9-489f-8801-4177768eadc1.png)
![2](https://user-images.githubusercontent.com/79224065/149330007-47f0efc5-78d2-46fe-b138-44aeca09c4fd.png)

**Comment by DaanGamesDG** [comment]
Oh no I mean when restarting the application, instead of "pm2 restart <id>" you have to use "pm2 restart <id> --update-env". Sorry for the confusion

**Comment by Mazeee1** [comment]
Ah, thanks.
So I did that and it seems like it made the error in the console go away but it still won't work.
When I put in the correct password it comes up with error 500, but when I put in an incorrect password it comes up with 404. Odd

![image](https://user-images.githubusercontent.com/79224065/149340516-c6f46b38-5a7c-4702-aee4-d54a1b2aad75.png)

**Comment by DaanGamesDG** [comment]
Hmmm, weird. Can you show me the pm2 logs?

**Comment by Mazeee1** [comment]
This seems to come up whenever I put my password in.
![image](https://user-images.githubusercontent.com/79224065/149341249-bbe1491c-ffce-4836-a247-cab84b97a372.png)

**Comment by DaanGamesDG** [comment]
Your Encryption key is too big, hoe many characters did you use?

**Comment by Mazeee1** [comment]
16. I'll shorten it.

**Comment by DaanGamesDG** [comment]
16 should be fine, try 8 and see if that works

**Comment by Mazeee1** [comment]
I don't mind starting again, and rebuilding my vps since there's nothing important on there.
I won't do it if it's not necessary though.

**Comment by DaanGamesDG** [comment]
I don't think it is necessary, can you re-generate one at: https://passwordsgenerator.net/plus/ using the following configuration: ![](https://cdn.daangamesdg.xyz/files/7pEX12at.png)

**Comment by Mazeee1** [comment]
That worked. Thanks so much :)

**Comment by Mazeee1** [comment]
That's great.
